### PR TITLE
Adjust sample command outputs in the Agent Fleet documentation

### PR DESCRIPTION
**Pull request by kunisen**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? 
  - Yes
- Have you followed the [contributor guidelines](https://github.com/elastic/cloud-on-k8s/tree/main/CONTRIBUTING.md)? 
  - Yes
- If you submit code, is your pull request against main? We recommend pull requests against main. We will backport them as needed. 
  - It's not code.

### Background:
- The manifest says `xxx-quickstart`, i.e. `fleet-server-quickstart`,
- however, the sample response says things without `-quickstart`…

image

### TL;DR:

This PR is to add `-quickstart` in sample response part, i.e. change pod name from `elastic-agent-agent-t49fd` to `elastic-agent-quickstart-agent-t49fd`, so as to keep the sample yaml manifest and the sample response aligned and easy to follow.

---

cc @thbkrkr as we chatted this internally. Thanks for your quick response! 🙏

### CODE REVIEW

**Comment by thbkrkr** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by thbkrkr — APPROVED** [review]
